src/shared/globalStyles.js

```diff
@@ -47,13 +53,48 @@ export const GlobalStyles = createGlobalStyle`
   h3,
   h4,
   h5,
-  h6 {
+  h6 {   
+    font-family: ${(props) => props.theme.global.fonts.serif};
     color: ${(props) => props.theme.global.colors.black};
-    font-family: 'Merriweather', serif;
     font-weight: bold;
-    line-height: 1.3;
+    line-height: 1.2;
+    margin : 0;
   }
 
+  h1 {
+    font-size: ${(props) => props.theme.post.fontSizes.xxl}
+  }
+
+  h2 {
+    font-size: ${(props) => props.theme.post.fontSizes.xxl}
+  }
+
+  h3 {
+    font-size: ${(props) => props.theme.post.fontSizes.lg}
+  }
+
+  h4 {
+    font-size: ${(props) => props.theme.post.fontSizes.md}
+  }
+
+  @media ${(props) => props.theme.global.mediaQueries.sm} {
+    h1 {
+      font-size: ${(props) => props.theme.post.fontSizes.xxl}
+    }
+
+    h2 {
+      font-size: ${(props) => props.theme.post.fontSizes.xl}
+    }
+
+    h3 {
+      font-size: ${(props) => props.theme.post.fontSizes.lg}
+    }
+
+    h4 {
+      font-size: ${(props) => props.theme.post.fontSizes.md}
+    }
+  }
```

Comment:
use props.theme.global.fontSizes

You will need to add the font size values for lg, xl, and xxl to the /Shared/GlobalStyles.js